blocks/small-manual-promo-block/features/small-manual-promo/default.jsx

```diff
@@ -60,7 +12,7 @@ const SmallManualPromo = ({ customFields = { showImage: true, showHeadline: true
   }
   return (
     <LazyLoad enabled={customFields.lazyLoad && !isAdmin}>
```

Comment:
For consistency with the other promo code and readability, can we change this to:
```suggestion
  const shouldLazyLoad = customFields?.lazyLoad && !isAdmin;
  if (shouldLazyLoad && isServerSide()) {
    return null;
  }
  return (
    <LazyLoad enabled={shouldLazyLoad}>
```

Reply:
Sorry, I can't get it to select the prior code here and it's annoying...

Reply:
yes, refactored 👍